content/docs/buildpacks/language-family-buildpacks/java.md

```diff
@@ -164,39 +176,79 @@ pack inspect-image samples/app --bom | jq '.local[] | select(.name=="jre") | .me
 
 #### Configuring JVM at Runtime
+
 The runtime JVM can be configured in two ways:
-1. Buildpack-provided runtime components including the Memory Calculator accept semantically named environment variables which are then used to derive `JAVA_TOOL_OPTIONS` flags. Examples include: 
+
+1. Buildpack-provided runtime components including the Memory Calculator accept semantically named environment variables which are then used to derive `JAVA_TOOL_OPTIONS` flags. Examples include:
     * `BPL_JVM_HEAD_ROOM`
     * `BPL_JVM_LOADED_CLASS_COUNT`
     * `BPL_JVM_THREAD_COUNT`
 2. Flags can be set directly at runtime with the `JAVA_TOOL_OPTIONS` environment variable. User-provided flags will be appended to buildpack-provided flags. If the user and a buildpack set the same flag, user-provided flags take precedence.
 
 See the [homepage][bp/bellsoft-liberica] for the Bellsoft Liberica Buildpack for a full set of configuration options.
 
+### Using Alternative JVM Vendors
+
+By default, the [Paketo Java buildpack][bp/java] will use the Liberica JVM. The following alternative JVM vendors may be substituted in place of Liberica's JVM.
+
+* [Paketo Amazon Corretto Buildpack][bp/amazon-corretto]
+* [Paketo Azul Zulu Buildpack][bp/azul-zulu]
+* [Paketo BellSoft Liberica Buildpack - Default][bp/bellsoft-liberica]
+* [Paketo Eclipse OpenJ9 Buildpack][bp/eclipse-openj9]
+* [Paketo GraalVM Buildpack][bp/graalvm]
+* [Paketo SapMachine Buildpack][bp/sap-machine]
+
+To use an alternative JVM vendor, you will need to set two `--buildpack` arguments to `pack build`, one for the alternative JVM vendor you'd like to use and one for the Paketo Java buildpack (in that order). This works because while you end up with two JVM vendor buildpacks, the first one, the one you're specifying will claim the buildplan entries so the second one will end up being a noop and doing nothing.
+
+This example will switch in the Azul Zulu buildpack:
+
+{{< code/copyable >}}
+pack build samples/jar --buildpack gcr.io/paketo-buidpacks/azul-zulu --buildpack paketo-buildpacks/java`
+{{< /code/copyable >}}
+
+There is one drawback to this approach. When using the method above to specify an alternative JVM vendor buildpack, this alternate buildpack ends up running before the CA certs buildpack and therefore traffic from the alternate JVM vendor buildpack won’t trust any additional CA certs. This is not expected to impact many users because JVM buildpacks should reach out to URLs that have a cert signed by a known authority with a CA in the default system truststore.
+
+If you have customized your JVM vendor buildpack to download the JVM from a URL that uses a certificate not signed by a well-known CA, you can workaround this by specifying the CA certs buildpack to run first. This works because while you will end up with the CA certificates buildpack specified twice, the lifecycle is smart enough to drop the second one.
```

Comment:
```suggestion
If you have customized your JVM buildpack to download the JVM from a URL that uses a certificate not signed by a well-known CA, you can workaround this by specifying the CA certs buildpack to run first. This works because while you will end up with the CA certificates buildpack specified twice, the lifecycle is smart enough to drop the second one.
```